sdk/servicebus/service-bus/test/README.md

```diff
@@ -105,7 +105,7 @@ Go through the following setup in order to delete and create the required servic
 
 **Assign owner role to the registered application**
 
-- In the azure-portal, go to your servicebus-namespace and assign **Azure Service Bus Data Owner (Preview)** role to the registered application.
+- In the azure-portal, go to your servicebus-namespace and assign **Azure Service Bus Data Owner** role to the registered application.
 - This can be done from `Role assignment` section of `Access control (IAM)` tab (in the left-side-navbar of your servicebus-namespace in the azure-portal)<br>
   _Doing this would allow the registered application manage the namespace, i.e., entity creation, deletion, etc.,_<br>
   _**Note:** AAD RBAC is enabled only on the new namespaces in this region for the preview. Please ensure that your servicebus-namespace is present in one of these regions: **US East, US East 2, or West Europe**. If not, create the namespace in one of those regions._
```

Comment:
Does this note need updating if it's no longer in preview? Are we still restricted to these 3 regions?

Reply:
Nice catch!

Reply:
Updated! Confirmed via e-mail thread that was shared by @HarshaNalluru